README.md

```diff
@@ -3,40 +3,85 @@
 > Official GitHub Action for [OSSF scorecard](https://github.com/ossf/scorecard).
 
 ## Installation
-The simplest and quickest way to install Scorecards's GitHub action is from the [GitHub's marketplace](https://github.com/marketplace/actions/ossf-scorecard-action).
-**WARNING**: the action does *not* support private repositories.
+**NOTE**: The Scorecards GitHub Action does not support private repositories. Private repositories can run Scorecards manually by following the [standard installation instructions)[https://github.com/ossf/scorecard#installation].
```

Comment:
so the news is that we support private repos:
1. running the action works if they follow the permission declaration as in https://github.com/test-organization-ls/action-test/blob/main/.github/workflows/scorecards-analysis.yml#L19
2. upload of the results to the scanning dashboard only work for repos that have GitHub Advanced Security (not under our control)

The use case of 1) would be for a user who wants to run scorecard and do some analysis on their own. Maybe we can ignore this part and just say that the action is supported for private repos with GitHub Advanced Security.

Reply:
Yeah, I agree, I doubt there's a lot of users falling into that use case. Let's keep it simple for now.